test/lib/common_test.js

```diff
@@ -834,3 +834,30 @@ exports.postgresql = {
     })
   }
 }
+
+// MongoDB
+exports.hypersecuredb = {
+  itCreatedServiceFilesWithExpectedContent: function (serviceName, serviceCredentials, servicePlan) {
+    var description = 'hypersecuredb'
+    var mapping = 'hypersecure_dbaas_mongodb'
+    var label = 'hypersecuredb'
```

Comment:
The label in the `unsupported_bluemix.json` is `"label": "hypersecuredbaas"` should this be `var label = 'hypersecuredbass` ?